static/js/lightbox.js

```diff
@@ -199,6 +201,27 @@ export function render_lightbox_list_images(preview_source) {
     }
 }
 
+export function init_image_title_tooltip(payload) {
+    const url = new URL(payload.source, window.location.href);
+    const file_name = url.search.slice(url.search.lastIndexOf("F") + 1, -10);
+
+    tippy(".image-description .title", {
+        arrow: true,
+        placement: "left",
+        onShow(instance) {
+            const elem = $(instance.reference);
+            const image_name = elem.text();
+            const content = ui_util.parse_html(`${image_name} <p>File name: ${file_name}</p>`);
```

Comment:
This opens up the code to cross site scripting attacks. In fact, you can test it yourself by putting html in the image name when you upload it:

![xss](https://user-images.githubusercontent.com/3046397/158451903-1cf58df9-4390-4772-a77b-508fcf3c2ef7.png)

Now imagine if you put some malicious javascript in the image name, and sent it to a large public channel on Zulip!